lib/init.coffee

```diff
@@ -10,6 +15,82 @@ module.exports =
       type: 'string'
       default: ''
 
-
   activate: ->
-    console.log 'activate linter-erlc'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-erlc.executablePath',
+    (executablePath) =>
+      @executablePath = executablePath
+  deactivate: ->
+    @subscriptions.dispose()
+  provideLinter: ->
+    helpers = require('atom-linter')
+    provider =
+      name: 'Erlc'
```

Comment:
`erlc` is all lowercase in [the documentation](http://erlang.org/doc/man/erlc.html) you should match that.